tests/lib/rules/use-isnan.js

```diff
@@ -183,6 +194,10 @@ ruleTester.run("use-isnan", rule, {
         "foo.lastIndexOf(NaN)",
         "foo.indexOf(Number.NaN)",
         "foo.lastIndexOf(Number.NaN)",
+        "foo.indexOf((NaN, 1))",
+        "foo.lastIndexOf((NaN, 1))",
+        "foo.indexOf((Number.NaN, 1))",
+        "foo.lastIndexOf((Number.NaN, 1))",
```

Comment:
These tests should probably have `options: [{ enforceForIndexOf: true }]`? Otherwise, they would be valid regardless of the position of NaN, because the rule doesn't check indexOf by default.

Reply:
you're totally right, I somehow missed that. thanks!